app/jinja_filters.py

```diff
@@ -66,6 +70,12 @@ def format_multilined_string(context, value):
     return result
 
 
+@blueprint.app_template_filter()
+def get_current_date(date_format='%d %B %Y'):
```

Comment:
%d will give a leading zero for days < 10 ? perhaps %-d may be used (haven't tested this)

Reply:
Good catch.

Looks like that's also incorrect on `format_datetime()`

The correct format is being used in `format_date()`

Please move these formats into variables to help prevent these sort of inconsistencies occurring.